test/onnx/torchlib/README.md

```diff
@@ -36,7 +36,7 @@ Sometimes, there is no existing OpInfo that fits our need to test an operator. Y
 
 Follow the steps below to create new OpInfo tests:
 
-1. Use the implementation for `ops.aten.slice_scatter` as a reference (https://github.com/microsoft/onnxscript/blob/e67335101e4a06b8cc98cb4129935a9af5062c77/tests/function_libs/torch_lib/extra_opinfo.py#L2412-L2418) to declare an OpInfo in [`extra_opinfo.py`](./extra_opinfo.py)
+1. Use the implementation for `ops.aten.slice_scatter` as a reference (https://github.com/microsoft/onnxscript/blob/e67335101e4a06b8cc98cb4129935a9af5062c77/tests/function_libs/torch_lib/extra_opinfo.py#L2412-L2418) to declare an `OpInfo` in [`opinfo/core.py`](../../../torch/testing/_internal/opinfo/core.py)
```

Comment:
```suggestion
1. Use the implementation for `ops.aten.slice_scatter` as a reference (https://github.com/microsoft/onnxscript/blob/e67335101e4a06b8cc98cb4129935a9af5062c77/tests/function_libs/torch_lib/extra_opinfo.py#L2412-L2418) to declare an `OpInfo` in [`extra_opinfo.py`](./extra_opinfo.py)
```

please revert this part of the change

Reply:
Yes that’s expected. We will be adding this file as part of a code migration so it will appear soon

Reply:
@justinchuby ok, i'll keep it w/o a link then for now to avoid CI check failures.